mibidata/tiff.py

```diff
@@ -21,22 +21,22 @@
 # Datetime format saved by TiffFile
 _DATETIME_FORMAT = '%Y:%m:%d %H:%M:%S'
 # Conversion factor from motor unit to cm
-_CM_FACTOR = 100000
+_MICRON_FACTOR = 100000
 # Max denominator for rational arguments in tifffile.py
 _MAX_DENOMINATOR = 1000000
 # Encoding of tiff tags.
 ENCODING = 'utf-8'
 
 
-def _motor_to_cm(arg):
-    """Converts motor unit (1cm = 1e5 motor units) to a fraction tuple in cm."""
-    frac = Fraction(arg, _CM_FACTOR).limit_denominator(_MAX_DENOMINATOR)
+def _micron_to_cm(arg):
+    """Converts microns (1cm = 1e5 microns) to a fraction tuple in cm."""
+    frac = Fraction(float(arg)/10000).limit_denominator(_MAX_DENOMINATOR)
     return frac.numerator, frac.denominator
 
 
-def _cm_to_motor(arg):
-    """Converts cm fraction to a motor unit (1cm = 1e5 motor units)."""
-    return int(np.round(float(arg[0]) / float(arg[1]) * _CM_FACTOR))
+def _cm_to_micron(arg):
+    """Converts cm fraction to microns (1cm = 1e4 microns)."""
+    return float(arg[0]) / float(arg[1]) * 10000
```

Comment:
Please use the named constant rather than a number here.

Reply:
done